packages/datepicker/src/react/index.js

```diff
@@ -314,8 +314,8 @@ class DatePicker extends React.Component {
           </div>
         </Halo>
         {state.isOpen && [
-          <Overlay onClick={this.handleOverlayClick} />,
-          <div {...styles.calendarContainer(allProps)}>
+          <Overlay onClick={this.handleOverlayClick} key="dialog-overlay" />,
```

Comment:
any better suggestion on key names?

Reply:
no, I think it works for me.  These don't have to be globally unique, right?  Just w/in component.  In that case, probably not a big deal.